feat: Add new "Recently Aired Shows" menu for tracking latest anime episodes

Alqatrony opened the pull request:
## 🎯 Summary
This PR introduces a **brand new feature**: "Recently Aired Shows" - a dedicated menu that displays all anime episodes that aired in the last 7 days.

## ✨ What's New
- New main menu item "Recently Aired Shows"
- Shows all anime episodes from the past 7 days
- Pagination support (24 items per page)
- Quality filtering to show only relevant content

## 📝 Implementation

### New Methods in `AniListBrowser.py`:
- `get_recently_aired_shows()` - Main entry point
- `get_recently_aired_shows_res()` - GraphQL query for airing schedules  
- `process_recently_aired_view()` - Processes and filters results

### New Route in `Main.py`:
- `RECENTLY_AIRED_SHOWS()` - Route handler
- Added menu item to main navigation

## 🚀 Features
- **7-Day Coverage**: Shows all episodes from past week
- **Quality Filtering**: Score ≥65 OR popularity ≥10,000
- **Pagination**: 24 items per page with Next Page button
- **Chronological**: Most recent episodes first

## 🐛 Bug Fix
- Fixed TypeError in `database_update_show()` date handling

## 🧪 Testing
- ✅ Menu appears in main navigation
- ✅ Shows 24 items per page
- ✅ Pagination works correctly
- ✅ Quality filtering working
- ✅ Episode numbers display properly

## 📸 Screenshots
<img width="1919" height="1079" alt="Screenshot 2025-09-13 213950" src="https://github.com/user-attachments/assets/bff93b95-6dff-4f26-8154-f3fbace87879" />

Alqatrony committed:
feat: Add new Recently Aired Shows menu

- Create new menu item for tracking recently aired anime episodes
- Implement 7-day window coverage (shows from past week)
- Add pagination support (24 items per page)
- Include quality filtering (score >= 65 OR popularity >= 10000)
- Fix TypeError bug in database_update_show date handling

Alqatrony committed:
feat: Add new Recently Aired Shows menu

- Create new menu item for tracking recently aired anime episodes
- Implement 7-day window coverage (shows from past week)
- Add pagination support (24 items per page)
- Include quality filtering (score >= 65 OR popularity >= 10000)
- Fix TypeError bug in database_update_show date handling

Goldenfreddy0703 commented:
Hey thank you so much for making this pr but we also need to apply this to Mal browser as well so that the mal people could have this. Also just a quick question, i did read into your pr but i have to ask, what is the point of this menu item if we have an airing anime calendar of course?

Reason why i ask is cause this menu items kinda operates the same as the calendar right?

Alqatrony committed:
feat: Add Recently Aired Shows feature with MAL support now Works with both AniList and MAL providers

Alqatrony committed:
feat: Implement native Jikan API for MAL Recently Aired Shows

- Replace AniList dependency with pure Jikan API implementation
- Use Jikan's native filtering: status=airing, min_score=6.5
- Sort by popularity (most popular airing shows)

Alqatrony commented:
## 🔄 Updated Implementation - Now with Native Jikan API

Thanks for the feedback! I've updated the MAL implementation to use Jikan API directly instead of converting AniList data.

### ✅ **What Changed:**
- **Complete API Separation**: MAL now uses pure Jikan API, AniList uses GraphQL
- **Native Jikan Filtering**: Uses `status=airing`, `min_score=6.5`, `order_by=popularity`
- **Better Reliability**: If one API goes down, the other works independently
- **Same UX**: Still 24 items per page with pagination

### 📊 **Result:**
Shows currently airing popular anime (score ≥6.5) for MAL users - actually more useful for discovering quality ongoing shows!

Ready for review! 🚀

Goldenfreddy0703 committed:
Refactor 'Recently Aired' routes and menu entries

Unified and renamed 'recently_aired_shows' to 'recently_aired' across codebase, including routes, menu items, and settings for all media types. Updated browser methods and removed legacy handler to streamline access and improve consistency for 'Recently Aired' content.

Alqatrony committed:
feat: implement true 7-day window for MAL recently_aired using Simkl monthly calendars

- Add monthly calendar JSON fetch methods to Simkl class (fetch_month_json, get_calendar_range)
- Update MalBrowser.get_recently_aired() to use monthly archives for month boundary support
- Add deduplication logic for raw entries across monthly calendar files
- Maintain existing MAL artwork pipeline and pagination
- Fix logging calls to use correct 'debug' level syntax
- Minor code cleanup and formatting improvements

Fixes issue where recently_aired missed episodes when 7-day window crossed month boundaries.

Alqatrony committed:
remove unused recently aired anime function

Alqatrony committed:
add recently aired routes for various media types

Goldenfreddy0703 committed:
Refactor animeschedule.py for compatibility and logging

Updated type annotations to be compatible with Python 3.8.15, improved API token loading to handle multiple database response formats, and added more detailed logging for token loading errors. Also replaced type hints with standard Python syntax for broader compatibility.
